mobile_verifier/src/banning/db.rs

```diff
@@ -0,0 +1,107 @@
+use std::collections::HashSet;
+
+use chrono::{DateTime, Duration, Utc};
+use file_store::mobile_ban::{BanAction, BanType, VerifiedBanReport};
+use futures::TryStreamExt;
+use helium_crypto::PublicKeyBinary;
+use sqlx::{PgConnection, PgPool, Row};
+
+use super::BAN_CLEANUP_DAYS;
+
+pub async fn get_banned_radios(
+    pool: &PgPool,
+    timestamp: DateTime<Utc>,
+) -> anyhow::Result<HashSet<PublicKeyBinary>> {
+    let banned = sqlx::query(
+        r#"
+            SELECT hotspot_pubkey
+            FROM hotspot_bans
+            WHERE 
+                ban_type in ('all', 'poc')
+                AND received_timestamp <= $1
+                AND (expiration_timestamp IS NULL 
+                    OR expiration_timestamp >= $1)
+            "#,
+    )
+    .bind(timestamp)
+    .fetch(pool)
+    .map_ok(|row| row.get("hotspot_pubkey"))
+    .try_collect()
+    .await?;
+
+    Ok(banned)
+}
+
+pub(super) async fn clear_bans(
```

Comment:
I would make `BAN_CLEANUP_DAYS` as input value of this function to make it more flexible and code more predictable.